src/modules/GroupChannel/context/GroupChannelProvider.tsx

```diff
@@ -291,6 +293,7 @@ const GroupChannelProvider = (props: GroupChannelContextProps) => {
         // Reset states when channel changes
         setQuoteMessage(null);
         setAnimatedMessageId(0);
+        scrollPubSub.publish('scrollToBottom', null);
```

Comment:
What if startingPoint is given? Should we not be scroll to it?

Reply:
This is a side effect triggered when the channelUrl is changed.

There is a separate side effect for the startingPoint below. ([link](https://github.com/sendbird/sendbird-uikit-react/blob/feat/collections/src/modules/GroupChannel/context/GroupChannelProvider.tsx#L318-L325))

Upon consideration, it seems we can execute this behavior in the clean-up of the [side effect for Scroll to bottom](https://github.com/sendbird/sendbird-uikit-react/blob/feat/collections/src/modules/GroupChannel/context/GroupChannelProvider.tsx#L298-L316).
let me change

Reply:
addressed in https://github.com/sendbird/sendbird-uikit-react/pull/927/commits/48d51f9f77f74442e7bc93268989f841ec8da484